handlers/proxy.go

```diff
@@ -88,6 +122,47 @@ func extractUrl(c *fiber.Ctx) (string, error) {
 	return urlQuery.String(), nil
 }
 
+// getFlareSolverrCookies retrieves cookies from FlareSolverr for the given URL
+func getFlareSolverrCookies(targetURL string) (string, error) {
+	if flareSolverrHost == "" {
+		return "", fmt.Errorf("FLARESOLVERR_HOST environment variable not set")
+	}
+
+	reqBody := FlareSolverrRequest{
```

Comment:
This is the only place you use `FlareSolverrRequest`, I don't know Go enough but maybe you can just inline it